chroma-manager/chroma_core/migrations/0023_auto__add_unconfigurersyslogjob__add_getcorosyncstatejob__add_pacemake.py

```diff
@@ -232,15 +240,17 @@ def forwards(self, orm):
         # Deleting field 'ManagedHost.corosync_reported_up'
         db.delete_column('chroma_core_managedhost', 'corosync_reported_up')
 
-        def create_action_record(model_name, state, host_id):
+        def create_action_record(model_name, table_dict):
             app_name = 'chroma_core'
             ct, _ = orm['contenttypes.ContentType'].objects.get_or_create(model=model_name.lower(),
                                                                           app_label=app_name,
                                                                           defaults=dict(name=model_name))
 
-            db.execute("insert into chroma_core_%s (state_modified_at, state, immutable_state, content_type_id, host_id) "
-                       "values ('%s', '%s', 'f', %s, %s)" % (model_name, datetime.datetime.now(), state, ct.id, host_id))
-
+            table_dict['content_type_id'] = ct.id
+            fields = ", ".join(map(str, table_dict.keys()))
+            values = ", ".join(map(str, map(lambda x: "'%s'" % x, table_dict.values())))
```

Comment:
couldn't you just get the items here and then unzip them?

Reply:
hmm, maybe. This takes something like:
```
{
   'state_modified_at': datetime.datetime.now(),
   'state', 'managed',
   'immutable_state': 'f',
   'host_id': '9'
}
```
and gives me:
```
fields = "state_modified_at, state, immutable_state, host_id"
values = "'2018-06-06', 'managed', 'f', '9'"
```
Perhaps I'm misunderstanding, but doesn't zip take two arrays and create a new list of tuples?